src/tun.c

```diff
@@ -20,6 +20,12 @@
 #include <string.h>
 #include <unistd.h>
 
+#include "exec.h"
```

Comment:
We should keep the original here --- including system header before local headers.

Which enums were being redefined?  Let's fix that in `ip.h`, rather than changing the include order.